.github/workflows/ci.yml

```diff
@@ -0,0 +1,146 @@
+name: CI
+
+on: [pull_request, push]
+
+jobs:
+  Build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    container:
+      image: elementary/docker:unstable
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install Depedencies
+        env:
+          dependency_packages: libvala-dev valac meson
+        run: apt-get update && apt-get -y install ${dependency_packages}
+
+      - name: Build
+        run: meson build && ninja -C build
+
+      - name: Upload
+        uses: actions/upload-artifact@v1
+        with:
+          name: vala-lint
+          path: ./
+
+  Test:
+    name: Test Build
+    runs-on: ubuntu-latest
+    needs: Build
+
+    container:
+      image: elementary/docker:unstable
```

Comment:
can just do `container: elementary/docker:unstable`

Reply:
`container` needs to be an object, not a string. You get both of these errors in the workflow visual editor on GitHub.

Reply:
I was just going off of the docs https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idcontainer

Reply:
Something is wrong then...

![Screenshot from 2019-10-11 11 09 19](https://user-images.githubusercontent.com/3385679/66670700-a188a280-ec17-11e9-882d-ff81dbeba0e6.jpeg)